src/c++/perf_analyzer/request_rate_manager.h

```diff
@@ -141,6 +145,9 @@ class RequestRateManager : public LoadManager {
       std::shared_ptr<ThreadStat>,
       std::shared_ptr<RequestRateWorker::ThreadConfig>);
 
+  // Determine number of threads based on sequence model and max threads
```

Comment:
again, don't say HOW. I don't think this needs a comment, as the WHAT is just "determines number of threads", which is what the function name is.